pkg/controller/recording/recording_controller.go

```diff
@@ -305,48 +310,48 @@ func (r *ReconcileRecording) getFlightRecorder(ctx context.Context, recording *r
 	return jfr, nil
 }
 
-func findDownloadURL(cjfr jfrclient.ContainerJfrClient, filename string) (*string, error) {
+func findURLs(cjfr jfrclient.ContainerJfrClient, filename string) (*string, *string, error) {
```

Comment:
To make this more readable, let's change the signature to something like this:
```suggestion
func findSavedRecording(cjfr jfrclient.ContainerJfrClient, filename string) (*jfrclient.SavedRecording, error) {
```
Then just return the whole `SavedRecording` instead of the separate URLs. It could lead to bugs if we mix them up int the future.